sui_programmability/examples/fungible_tokens/README.md

```diff
@@ -2,6 +2,8 @@
 
 * MANAGED: a token managed by a treasurer trusted for minting and burning. This is how (e.g.) a fiat-backed stablecoin or an in-game virtual currency would work.
 * BASKET: a synthetic token backed by a basket of other assets. This how (e.g.) a [Special Drawing Rights (SDR)](https://www.imf.org/en/About/Factsheets/Sheets/2016/08/01/14/51/Special-Drawing-Right-SDR)-like asset would work.
-* regulated_coin: a coin managed by a central authority which can freeze accounts
+* REGULATED_COIN: a coin managed by a central authority which can freeze accounts
+* PRIVATE_COIN: a coin which has the option of hiding the amount that you transact
```

Comment:
@randall-Mysten please fix the grammar issue on this line:
https://www.grammarly.com/blog/using-that-and-which-is-all-about-restrictive-and-non-restrictive-clauses/

Reply:
Thanks Clay, fixed!